src/common/utility.js

```diff
@@ -362,3 +362,51 @@ export const getBaseRef = (ref) => {
     // in other cases they are in ref.current
     return ref?.current?.base?.style ? ref?.current?.base : ref?.current
 }
+
+const redirect = (subdomain) => {
+    const redirection_url = `${subdomain}.deriv.com`
+    window.location.host = redirection_url
+}
+
+const redirectToDeriv = (full_domain) => {
+    window.location.host = full_domain.includes('staging') ? 'staging.deriv.com' : 'deriv.com'
+}
+
+export const handleDerivRedirect = (country, subdomain) => {
+    if (country === 'nl') {
+        redirect(subdomain.includes('staging') ? subdomain : 'eu')
+    } else if (country === 'gb') {
+        redirect(subdomain.includes('staging') ? subdomain : 'uk')
+    }
+}
```

Comment:
I'd suggest we have an array either inside the function or as a global constant such as:
```jsx
const eu_subdomain_countries = ['nl']
const uk_subdomain_countries = ['gb']
```
and then doing the if checks against those arrays so that it will be easier to extend when we need to add the countries to it.